castle/cms/behaviors/versionable.py

```diff
@@ -0,0 +1,103 @@
+from plone.app.versioningbehavior import _
+from plone.autoform import directives as form
+from plone.autoform.interfaces import IFormFieldProvider
+from plone.dexterity.interfaces import IDexterityContent
+from plone.supermodel import model
+from z3c.form.interfaces import IAddForm
+from z3c.form.interfaces import IEditForm
+from zope import schema
+from zope.annotation.interfaces import IAnnotations
+from zope.component import adapter
+from zope.interface import alsoProvides
+from zope.interface import implementer
+from zope.interface import Interface
+from zope.publisher.interfaces.browser import IBrowserRequest
+
+
+class IVersionable(model.Schema):
```

Comment:
Since the changenote is the only thing that changed from plone.app.versioningbehavior, I'm wondering if you could do something like this (obviously with # noqa:F401 on imports that are imported but not used in the file, and maybe not even all of those would need to be imported, idk): 

`from plone.app.versioningbehavior.behaviors import IVersionable as IOldVersionable`
`from plone.app.versioningbehavior.behaviors import IVersioningSupport`
`from plone.app.versioningbehavior.behaviors import Versionable`
`from plone.app.versioningbehavior.utils import get_change_note`

`class IVersionable(IOldVersionable):`
`    changeNote = schema.TextLine(`
`       title=_(u'label_change_note', default=u'Change Note'),`
`        description=_(u'help_change_note',`
`                     default=u'Enter a comment that describes the changes you made.'),`
`        required=True)`
`alsoProvides(IVersionable, IFormFieldProvider)`

I'm not 100% certain it would work, but it seems on the face of it that it would. Having a bunch less code in our codebase seems desirable to me, but I'm happy to have @zombified  tell me I'm wrong or it doesn't much matter.